docs/html/conf.py

```diff
@@ -55,6 +55,7 @@
 # -- Options for myst-parser ----------------------------------------------------------
 
 myst_enable_extensions = ["deflist"]
+myst_heading_anchors = 3
```

Comment:
How is this related?

Reply:
IIUC, this was for a documentation-related issue -- the CI was failing back on a docs issue back when this PR was filed.

Reply:
I've rebased this, which should have removed this commit.

Reply:
OK, let's remove it from here and if the docs still fail, put it in its own PR. But presumably it's no longer needed if main is passing CI now.